lib/danger/available_values.rb

```diff
@@ -0,0 +1,27 @@
+module Danger
+  # Defines all the values that should be available in someone's Dangerfile
+  class AvailableValues
+    def self.all
+      self.scm + self.request_source
+    end
+
+    def self.scm
+      [
+        :lines_of_code,
+        :files_modified,
+        :files_removed,
+        :files_added,
+        :deletions,
+        :insertions
+      ]
+    end
+
+    def self.request_source
+      [
+        :pr_title,
+        :pr_body,
+        :pr_author
+      ]
+    end
```

Comment:
I like these ideas! Ideally there should be a way to apply rubydoc so we can generate documentation from this file for the README too